app/src/main/java/com/github/ybecker/epforuml/chat/SearchActivity.kt

```diff
@@ -0,0 +1,81 @@
+package com.github.ybecker.epforuml.chat
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.*
+import androidx.appcompat.app.AppCompatActivity
+import com.github.ybecker.epforuml.MainActivity
+import com.github.ybecker.epforuml.R
+import com.github.ybecker.epforuml.database.DatabaseManager.db
+import com.github.ybecker.epforuml.database.DatabaseManager.user
+
+
+/**
+ * This activity represents a search view taking a list and treating the results as asked
+ * In the future, I would like to make Generic so that it can be reused with other lists
+ */
+class SearchActivity : AppCompatActivity() {
+
+    private lateinit var list: ArrayList<String>
+
+    private lateinit var listView: ListView
+    private lateinit var listAdapter: ArrayAdapter<String>
+    private lateinit var searchView: SearchView
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_search)
+        listView = findViewById(R.id.listView)
+        searchView = findViewById(R.id.searchView)
+
+        val button: Button = findViewById(R.id.back_to_home_button)
```

Comment:
Instead of implementing this button, enable the ActionBar in this activity to allow the integrated back button to be displayed. Then you simply have to specify which activity it should lead back to.